File: models/node.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace mind.map.models
{
    public class node
    {
        public node(string name, string description) {
            Name = name;
            Description = description;
        }
        public string Name { get; }
        public string Description { get; }
    }
}

File: ui/app/models/node.ts
/**
 * Created by appel on 12/13/2016.
 */

export class Node implements Nod{

  // Name: string;
  // Description: string

  constructor(public Name: string, public Description: string){
  }
 }

interface Nod{
  Name: string;
  Description: string

}
